pkg/cmd/roachtest/test_runner.go

```diff
@@ -716,6 +720,7 @@ func (r *testRunner) runWorker(
 		// case.
 		if err := VerifyLibraries(testToRun.spec.NativeLibs, arch); err != nil {
 			shout(ctx, l, stdout, "Library verification failed: %s", err)
+			c.Destroy(context.Background(), closeLogger, l)
```

Comment:
I'm still fuzzy why we're adding `c.Destroy` to error paths? Don't we want to fail fast instead and let GC handle the clean-up at a later time?